src/utils/is-object.ts

```diff
@@ -0,0 +1,3 @@
+const isObject = (value: any): boolean => value !== null && typeof value === 'object';
```

Comment:
may be **isPlainObject** is more clear name for this function?

Reply:
Also, this function seems problematic, ie **Array**, **Function** , "Date" etc are also treated as object in js

Reply:
Nice catch. I've updated the function name to be `isObjectLike` and added a condition ensuring it's not an array. Since this function only deals with serializable data, this should be sufficient.